test/Sentry.Tests/Internals/JsonTests.cs

```diff
@@ -175,37 +175,6 @@ public void WriteDynamicValue_ClassWithAssembly_SerializedClassWithNullAssembly(
         Assert.Equal(expectedSerializedData, serializedString);
     }
 
-    [Fact]
```

Comment:
Note, I deleted this test because it behaves differently on android, but also because we don't ever serialize a `TimeZoneInfo` in this way.  We serialize only the `Id` and `DisplayName` - separately.